services/brig/src/Brig/User/Search/SearchIndex.hs

```diff
@@ -107,8 +121,12 @@ defaultUserQuery u teamSearchInfo (normalized -> term') =
           boolQuery
             { ES.boolQueryMustMatch =
                 [ ES.QueryBoolQuery
-                    boolQuery {ES.boolQueryShouldMatch = [matchPhraseOrPrefix, legacyPrefixMatch]}
+                    boolQuery
+                      { ES.boolQueryShouldMatch = [matchPhraseOrPrefix, legacyPrefixMatch],
+                        ES.boolQueryMustNotMatch = [termQ "handle" term']
+                      }
                 ],
+              -- This removes exact handle matches, as they are fetched from cassandra
```

Comment:
```suggestion
                    boolQuery
                      { ES.boolQueryShouldMatch = [matchPhraseOrPrefix, legacyPrefixMatch],
                        -- This removes exact handle matches, as they are fetched from cassandra
                        ES.boolQueryMustNotMatch = [termQ "handle" term']
                      }
                ],
```

The comment is at wrong place, either I was being absent minded or ormolu did something weird!